modules/win-instance/watchmaker-win-instance.template.cfn.yaml

```diff
@@ -434,6 +448,61 @@ Resources:
                       "log_stream_name": "default_logs_{instance_id}"
                     }
                   }
+            'c:\cfn\scripts\AmazonCloudWatchAgent\install-python.ps1':
```

Comment:
Same comment as the autoscale template.

Reply:
Converted the windows cwl config manipulation to powershell and removed this quirk